Code/GraphMol/FileParsers/MolSGroupParsing.cpp

```diff
@@ -470,7 +535,16 @@ void ParseSGroupV2000PXALine(IDX_TO_SGROUP_MAP &sGroupMap, RWMol *mol,
   unsigned int pos = 6;
 
   unsigned int sgIdx = ParseSGroupIntField(text, line, pos);
```

Comment:
I made a mistake here when I implemented the SGroups: the leading number in the field's data is not an sgIdx, but an AtomIdx.

I didn't understand the spec on these were well, and I didn't have any sample files with PXA lines to help me understand them. So, I was actually not going to implement PXA, but looks that at some point I did, in the wrong way.

Reply:
Ah, that explains it.
Ok, I updated the PR, moved that chunk of parsing code to `MolFileParser.cpp`, and now have it just set an atom property.